demisto_sdk/commands/common/tools.py

```diff
@@ -2431,7 +2431,7 @@ def get_demisto_version(client: demisto_client) -> Version:
         return Version(Version(about_data.get("demistoVersion")).base_version)
     except Exception:
         logger.warning(
-            "Could not parse Xsoar version, please make sure the environment is properly configured."
+            "Could not parse server version, please make sure the environment is properly configured."
```

Comment:
@Shellyber perhaps we should halt the run, instead of just warning

Reply:
We are halting the run. In the upload function below (see images in the PR body).
Here we didn't touch the code, and for this use-case I don't think it worth changing the logic here